docs/git/cheatsheet.md

```diff
@@ -0,0 +1,135 @@
+---
+id: index
+title: Github Desktop Cheatsheet
+sidebar_label: Github Desktop Cheatsheet
+---
+
+## I want to get code from a repo onto my computer (Cloning)
+
+Follow these steps if you need to get code from a Github repo onto your computer (for example to complete your coursework).
+
+:::info
+Github Desktop might look a bit different if are cloning the first repository on your computer.
+:::
+
+#### Video
+
+<video width="100%" controls>
+  <source src={require('./assets/cloning-video.mp4').default} />
+</video>
+
+#### Screenshots
+
+1. Open the repo on Github in your browser (for the coursework this will likely be https://github.com/CodeYourFuture/COURSEWORK_REPO_NAME):
+2. Click the "Code" button, which opens a menu:
+
+![Code menu](./assets/cloning-1.png)
+
+3. Copy the URL (Pro Tip: you can click the clipboard icon on the right to copy automatically!)
+4. Open Github Desktop, and click on the Current Repository menu in the top left:
+
+![Current repository menu](./assets/cloning-2.png)
+
+5. Click the Add button at the top, then the Clone Repository option:
+
+![Add repo button](./assets/cloning-3.png)
+
+6. In the Clone Repository modal, make sure you are on the URL tab. Then paste the URL that you copied before into the "URL or username/repository" input:
+
+![Clone repository modal](./assets/cloning-4.png)
+
+7. Click the Clone button, and wait until it has finished cloning
```

Comment:
Let's add a step which explains what the second box (where to clone to) is, and that the default is probably fine.